6/main.js

```diff
@@ -0,0 +1,65 @@
+function thesmallestnumber(num1, num2, num3, num4) {
+  if (isNaN(Math.min.apply(null, [num1, num2, num3, num4]))) {
+    return "Введіть хоча б 4 числа.";
+  } else {
+    return Math.min.apply(null, [num1, num2, num3, num4]);
+  }
+}
+console.log(thesmallestnumber());
+function greatestcommonden(x = "Введіть хоча б 1 число.", y) {
```

Comment:
`x` - should not have a default value as a string. If you need the default one - create it inside the function. x - is an argument, which should be treated as a number only
```suggestion
function greatestCommondDivider(x, y) {
```